download.py

```diff
@@ -53,6 +54,20 @@ def download_run_fasterq_dump(accession, term, total_spots, output_directory, *,
     return correctness
 
 
+def _force_ena_http(url):
+    # FTP URLs from ENA do NOT currently include the scheme. Just prepend http://
+    # https://ena-docs.readthedocs.io/en/latest/retrieval/file-download.html
+    return 'http://' + url
+
+
+def _download_file(url, output_file, chunk_size=10**6):
+    with requests.get(url, stream=True) as response:
+        response.raise_for_status()
+        with output_file.open('wb') as file:
+            for chunk in response.iter_content(chunk_size):
+                file.write(chunk)
+
+
 def download_run_ftp(accession, term, out):
```

Comment:
What do you think about taking this code to a separate function? As @asuragan13 did in his PR

```python
url = (
        f"https://www.ebi.ac.uk/ena/portal/api/filereport?accession={accession}"
        "&result=read_run&fields=fastq_ftp,fastq_md5&format=json"
    )
    response = requests.get(url)
    ftps = response.json()[0]['fastq_ftp'].split(';')
    md5s = response.json()[0]['fastq_md5'].split(';')
```

Reply:
I'll alter my PR to use his implementation after it's merged.